components/IssueCard.vue

```diff
@@ -0,0 +1,15 @@
+<script lang="ts" setup>
+defineProps<{ id: string; title: string }>()
+</script>
+
+<template>
+  <li
+    class="flex flex-col gap-1 rounded-[4px] bg-rad-background-float px-2 pb-6 pt-2 hover:bg-rad-fill-float-hover"
+  >
+    <span class="text-xs font-medium text-rad-foreground-dim">
```

Comment:
Let's make that a `<small><pre>` for more semantic HTML.

Reply:
👍